src/serverMain/kotlin/com/studo/campusqr/auth/LdapAuth.kt

```diff
@@ -40,7 +40,7 @@ class LdapAuth(private val ldapUrl: String) : AuthProvider {
       ldapGroupRegex = getConfig("ldapGroupRegex")
       ldapPrintDebugLogs = getConfig("ldapPrintDebugLogs")
       ldapTimeoutMs = getConfig("ldapTimeoutMs")
-      ldapDefaultUserType = UserType.valueOf(getConfig("ldapDefaultUserType"))
+      ldapDefaultUserRole = UserRole.valueOf(getConfig("ldapDefaultUserType"))
```

Comment:
We should make this multi-role based. just split by ",", and do a trim on all of them.